src/evaluators/source-4-3.js

```diff
@@ -817,10 +818,18 @@ function parse_and_run(str) {
         (val, next_alternative) => {
             announce_output(output_prompt);
```

Comment:
Perhaps we need not print the output, since we're already setting the global variable?
If we print the output, there's a bit of duplication as both the value and the printed output are displayed:
![image](https://user-images.githubusercontent.com/35621759/76855892-0f9f5880-688d-11ea-94c1-114d4d2f05d7.png)

Reply:
I've removed the output printing

Reply:
Nice improvement!